apps/studio/src/lib/db/clients/sqlite.ts

```diff
@@ -60,14 +58,14 @@ const SD = SqliteData;
 export class SqliteClient extends BasicDatabaseClient<SqliteResult> {
 
   version: SqliteResult;
-  database: string;
+  databasePath: string;
 
   constructor(server: IDbConnectionServer, database: IDbConnectionDatabase) {
-    super(knex, sqliteContext);
+    super(knex, sqliteContext, server, database);
 
     this.dialect = 'sqlite';
-    this.dbReadOnlyMode = server?.config?.readOnlyMode || false;
-    this.database = database?.database;
+    this.readOnlyMode = server?.config?.readOnlyMode || false;
```

Comment:
NIT: I see this repeated a lot, probably means it should be in `BasicDatabaseClient`